### PR TITLE
Announcements uses mock data

### PR DESCRIPTION
**Pull request by Qurea2166**

### CODE REVIEW

File: classes/calendar.py [hunk 1]
```diff
@@ -1,6 +1,18 @@
 from uuid import uuid4
 
 class Calendar():
+  def __init__ (self, calendar_posts, event):
+    self._uuid = str(uuid4())
+    self._calendar_post = calendar_post
+    self._event = event
+    
+
+class Calendar_post(Calendar):
```
Comment (rileycassidy) [inline classes/calendar.py:10]: New calendar_post class?

**Review by rileycassidy — COMMENTED** [review]
Just saw the one weird thing, Announcement/Announcement.py seems to be fine

File: announcements/announcements.py [hunk 2]
```diff
@@ -1,32 +1,35 @@
 import domControl
-
+import crud_announcement
```
Comment (rileycassidy) [inline announcements/announcements.py:2]: from crud import announcement, crud_announcement does not exist

 or import crud.announcement would also be valid here

File: announcements/announcements.py [hunk 3]
```diff
@@ -1,32 +1,35 @@
...
   </div>
   '''
   
   return announcement_content
   
 
-# amount of announcements currently
-num_of_announcements = 5
+# get announcement list from announcement crud
+announcement_list = crud_announcement.get_existing_data("announcement.json")
```
Comment (rileycassidy) [inline announcements/announcements.py:23]: Same problem

File: classes/calendar.py (hunk 1 above)
Comment (rileycassidy) [inline classes/calendar.py:10]: I'm not sure when this was added but it looks broken...

**Review by rileycassidy — CHANGES_REQUESTED** [review]
File Pathing issue, with this everything else looks good

**Comment by mckinleyevans-wrdsb** [comment]
@Qurea2166 It looks like we doubled up on some work with Manahil's work that is already merged.  You'll need to do some conflict fixing

File: announcements/announcements.py (hunk 2 above)
Comment (Qurea2166) [reply announcements/announcements.py:2]: crud_announcement is actually a file we fetch in the index.html for the announcements

**Review by Harman0909 — APPROVED** [review]

**Review by rileycassidy — APPROVED** [review]

**Comment by mckinleyevans-wrdsb** [comment]
Cancelling this PR - repeats work that was already merged into main (hashtag my bad), also alters a number of files unrelated to this change

**Pull request closed without merging**